sdk/python/client.py

```diff
@@ -1,51 +1,130 @@
 import grpc
 import codegen.oomd_pb2
 import codegen.oomd_pb2_grpc
+import time
+from subprocess import Popen
+
 
-# Convert google.protobuf.pyext._message.MessageMapContainer object to Python dictionary
 def map_container_to_dict(map_container):
-    return dict({k: getattr(v, v.WhichOneof('kind')) for k, v in map_container.items()})
+    return dict({k: getattr(v, v.WhichOneof("kind")) for k, v in map_container.items()})
+
 
 class Client(object):
-    def __init__(self):
-        pass
+    def __init__(self, port: str, config_path: str):
+        try:
+            self.oomd = Popen(["oomd", config_path])
+        except e:
+            raise e
+
+        # wait oomd started
+        time.sleep(2)
+        channel = grpc.insecure_channel("localhost:%d" % port)
```

Comment:
what if the channel is disconnected halfway? Is there a way to recover?

If not, I would prefer leaving the channel and stub initialization to each method call.

Reply:
sound good, done.